test/test_env_setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
```

Comment:
Add `set -ex`, so any errors will end the script and commands that are run are printed prefixed with a `+`, so people know what is happening behind the scene (https://explainshell.com/explain?cmd=set+-ex)